server/helpers/dummyData/ingredients.js

```diff
@@ -0,0 +1,62 @@
+export default () => {
+  const ingredient = {
+    id: 1,
+    name: 'Onions'
+  };
+  const ingredients = {
```

Comment:
Why didn't you make use of an array instead?

Reply:
fixed https://github.com/hephzaron/youth-interact/pull/1/commits/748a6e90e4db20f9339f6b9911ebf3ee64a589b4